src/test/spec/crud.rs

```diff
@@ -18,6 +18,12 @@ fn test_predicate(test: &TestCase) -> bool {
     // The Rust driver doesn't support unacknowledged writes.
     let lower = test.description.to_lowercase();
 
-    // TODO: RUST-1071, RUST-1215: unskip comment tests
-    !lower.contains("unacknowledged") && !lower.contains("comment")
+    // TODO: RUST-1071, RUST-663: unskip comment tests
+    // and aggregate $out and $merge tests
+    !lower.contains("unacknowledged")
+        && (!lower.contains("comment")
+            || (lower.contains("estimated")
```

Comment:
Would this work as `(!lower.contains("comment") || lower.contains("estimateddocumentcount"))`?